sdk/src/main/java/com/mongodb/stitch/android/StitchError.java

```diff
@@ -23,10 +22,10 @@
      * @param error The network error.
      * @return An exception describing the network error.
      */
-    static StitchRequestException parseRequestError(final VolleyError error) {
+    static StitchException.StitchRequestException parseRequestError(final VolleyError error) {
```

Comment:
[q] Why not keep the static import?

Reply:
Must have gotten auto corrected by the IDE.